src/game/client/components/hud.cpp

```diff
@@ -1402,41 +1402,19 @@ void CHud::RenderMovementInformationTextContainer(STextContainerIndex &TextConta
 	}
 }
 
-void CHud::RenderMovementInformation()
+void CHud::GetMovementInformation(int ClientId, vec2 &DisplayPos, vec2 &DisplaySpeed, float &DisplayAngle)
```

Comment:
Return an object instead of using multiple output parameters:

```suggestion
CHud::CMovementInfo CHud::GetMovementInformation(int ClientId)
```

And can this function be `const`?

Reply:
I don't like objects for such small things, but if you prefer it sure

And yes can be made const